### DISCUSSION TITLE
segmentation at collaborators

### DISCUSSION THREAD

**Discussion by ayyaz-azeem** [FETS-AI/Challenge#145] [General]
which function (function name) is run on individual collaborators for segmentation that dispatches parameters to the aggregator function?

**Comment by sarthakpati** [comment]
Duplicate of #142, tagging @psfoley @alexey-gruzdev for answer there.

**Comment by dskhanirfan** [comment]
As the function [Collaborator.do_task()](https://github.com/intel/openfl/blob/develop/openfl/component/collaborator/collaborator.py#L184) - is used for Local task execution **line 188** mentions the function name that should be executed on the participating collaborator in the [plan.yaml](https://github.com/FETS-AI/Challenge/blob/main/Task_1/openfl-workspace/fets_challenge_workspace/plan/plan.yaml) and [FeTSChallengeModel](https://github.com/FETS-AI/Challenge/blob/main/Task_1/openfl-workspace/fets_challenge_workspace/impl/fets_challenge_model.py) is called for image segmentation?

Can you confirm if this function call sequence is correct?  @psfoley @alexey-gruzdev @sarthakpati @ayyaz-azeem
